# ---------------- /AgentArray.msg -------------------
# Custom ROS message for vehicle stats information
carla_circle/Agent[] agents
